registry/api/v2/descriptors.go

```diff
@@ -374,7 +346,7 @@ var routeDescriptors = []RouteDescriptor{
 									ContentType: "application/json; charset=utf-8",
 									Format:      errorsBody,
 								},
-								ErrorCodes: []ErrorCode{
+								ErrorCodes: []errcode.ErrorCode{
```

Comment:
No, what I am saying is that multiple errorcodes may not be needed. This still needs a defined status code for the response descriptor.

I think I may be wrong, since an endpoint may have different error codes. Albeit, the design here is to have a response descriptor per failure case. I don't recommend making these changes for this PR.